File: sofd-core-opus-integration/src/main/java/dk/sofd/opus/service/model/Affiliation.java
package dk.sofd.opus.service.model;

import java.util.Map;
import java.util.Set;

import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@EqualsAndHashCode(exclude = { "uuid" })
@Builder
@ToString
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.ALWAYS)
public class Affiliation {
    private String uuid;
    private String personUuid;
    private String master;
    private String masterId;
    private String startDate;
    private String stopDate;
    private boolean deleted;
    private String orgUnitUuid;
    private String employeeId;
    private String employmentTerms;
    private String employmentTermsText;
    private String payGrade;
    private String wageStep;
    private Double workingHoursDenominator;
    private Double workingHoursNumerator;
    private String affiliationType;
    private String positionId;
    private String positionName;
    private String positionTypeId;
    private String positionTypeName;
    private Set<String> functions;
    private Set<String> managerForUuids;
    private Map<String, String> localExtensions;
}

File: sofd-core-safety-net/Services/Sofd/Model/Affiliation.cs
using System;
using System.Collections.Generic;

namespace sofd_core_safety_net.Services.Sofd.Model
{
    public class Affiliation
    {
        public DateTime? StartDate { get;  set; }
        public DateTime? StopDate { get;  set; }
        public string PositionName { get;  set; }
        public string OrgunitUuid { get;  set; }
        public string Master { get;  set; }
        public string EmployeeId { get; set; }
        public bool Prime { get;  set; }
        public bool Deleted { get;  set; }
        public string OrgUnitUuid { get;  set; }
        public List<string> Functions { get; set; }
    }
}
